frodo.js

```diff
@@ -1,15 +1,16 @@
 /**
```

Comment:
This hardcoded value might be better suited as a constant at the top of the file.